src/nxdoc/nuxeo-server/administration/monitoring-and-maintenance/hotfixes-installation-notes-for-nuxeo-platform-lts-2019.md

```diff
@@ -92,6 +92,16 @@ Registration tokens are valid until your current contract's expiration date. Whe
 
 If you have any questions, feel free to contact our support team via a dedicated support ticket.
 
+## Hotfix 47
+
+### Manual replacement of nuxeoctl
+
+The hotfix 47 brings a fix to hide password in nuxeoctl.log when nuxeoctl is used to encrypt properties thanks to [NXP-30265](https://jira.nuxeo.com/browse/NXP-30265). But the hotfix mechanism cannot replace a file being run or change permissions on a file. Because of that, people who wants to benefit from this fix must replace it manually by downloading it from [NXP-30265](https://jira.nuxeo.com/browse/NXP-30265):
```

Comment:
```suggestion
The hotfix 47 brings a fix to hide passwords in nuxeoctl.log when `nuxeoctl` is used to encrypt properties, thanks to [NXP-30265](https://jira.nuxeo.com/browse/NXP-30265). Yet, the hotfix mechanism cannot replace a file being run or change permissions on a file. Because of that, people who want to benefit from this fix must replace it manually by downloading it from [NXP-30265](https://jira.nuxeo.com/browse/NXP-30265):
```